### PR TITLE
Fix classes toggling even with useStickyClasses set to false

### LINKED ISSUE #197: useStickyClasses False still adding classes
Using 
`stickybits('.sticky');`
Or 
`stickybits('.sticky', { useStickyClasses: false });`
Still adds classes and changes classes when scrolling.

This is happening on Mac Chrome (63.0.3239.132)
It is not happening on FireFox.

Using @version v2.1.1

### PR DESCRIPTION
**Pull request by hacknug**
This PR should fix the problem described in #197.
The bug was introduced by yours truly with #191.

Closes #197.

### CODE REVIEW

**Comment by yowainwright** [comment]
@hacknug will look asap! Thank you! 

Also, I will try to take a little time to improve acceptance tests soon ....and note your work in the readme. 🙏

File: src/stickybits.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ function Stickybits(target, obj) {
     -  uses a computed (`.definePosition()`)
     -  defined the position
   */
-  p.positionVal = this.definePosition() || 'fixed'
```
Comment (yowainwright) [inline src/stickybits.js]: `.definePosition()` defines the position. Defining the position (whether it is `sticky` or `fixed`) should happen within the `.definePosition()`.

Comment (hacknug) [reply src/stickybits.js]: So this change should be included in the new PR?

**Comment by yowainwright** [comment]
@hacknug 

1. I am going to give you a shout out in the readme. Your work is appreciated. 🙏
2. I'm going to write some more tests so that we can hopefully avoid this recent issue.
3. We can then address your original issue about the `.cleanup` method.

Would you mind closing this PR and fast following with the `.cleanup` method [issue](https://github.com/dollarshaveclub/stickybits/issues/188)?

Thank you!

**Pull request closed without merging**